### DISCUSSION TITLE
(Curso Acesso à dados com .NET, C#, Dapper e SQL Server) -(Aula OneToOne Part 2)

### DISCUSSION THREAD

**Discussion by jeannfrs38** [balta-io/forum#706] [Tira Dúvidas] [answered]
codigo da erro quando ultilizo o splitOn: "[Id]".
Quando eu tiro o splitOn funciona normalmente nao to conseguindo ver onde estou errando repeti a aula mas nao achei o erro.

![Captura de tela 2024-11-19 152306](https://github.com/user-attachments/assets/b1693cf9-424f-4da4-8ef1-b64906658ede)
![Captura de tela 2024-11-19 152318](https://github.com/user-attachments/assets/53a5dedb-15c7-4c67-bde3-85b7ba36b2cc)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @jeannfrs38, consegue comparar com o código do curso para ver o que pode estar dando de errado?
